wbooks/src/app/index.tsx

```diff
@@ -1,100 +1,18 @@
-/* eslint-disable no-use-before-define */
-/**
- * Sample React Native App
- * https://github.com/facebook/react-native
- *
- * Generated with the TypeScript template
- * https://github.com/react-native-community/react-native-template-typescript
- *
- * @format
- */
-
 import React from 'react';
-import { SafeAreaView, StyleSheet, ScrollView, View, Text, StatusBar } from 'react-native';
-import {
-  Header,
-  LearnMoreLinks,
-  Colors,
-  DebugInstructions,
-  ReloadInstructions
-} from 'react-native/Libraries/NewAppScreen';
+import { View } from 'react-native';
+import { BOOKS_MOCK } from '@constants/mockBooks';
+
+import BookCard from './screens/BookCard';
+import styles from './styles';
 
-const App = () => {
+function App() {
   return (
-    <>
-      <StatusBar barStyle="dark-content" />
-      <SafeAreaView>
-        <ScrollView contentInsetAdjustmentBehavior="automatic" style={styles.scrollView}>
-          <Header />
-          <View style={styles.body}>
-            <View style={styles.sectionContainer}>
-              <Text style={styles.sectionTitle}>Step One</Text>
-              <Text style={styles.sectionDescription}>
-                Edit <Text style={styles.highlight}>App.tsx</Text> to change this screen and then come back to
-                see your edits.
-              </Text>
-            </View>
-            <View style={styles.sectionContainer}>
-              <Text style={styles.sectionTitle}>See Your Changes</Text>
-              <Text style={styles.sectionDescription}>
-                <ReloadInstructions />
-              </Text>
-            </View>
-            <View style={styles.sectionContainer}>
-              <Text style={styles.sectionTitle}>Debug</Text>
-              <Text style={styles.sectionDescription}>
-                <DebugInstructions />
-              </Text>
-            </View>
-            <View style={styles.sectionContainer}>
-              <Text style={styles.sectionTitle}>Learn More</Text>
-              <Text style={styles.sectionDescription}>Read the docs to discover what to do next:</Text>
-            </View>
-            <LearnMoreLinks />
-          </View>
-        </ScrollView>
-      </SafeAreaView>
-    </>
+    <View style={styles.wrapper}>
+      {BOOKS_MOCK.map(book => (
+        <BookCard imageUrl={book.imageUrl} title={book.title} author={book.author} />
+      ))}
+    </View>
```

Comment:
2 things here:
* Use `SafeAreaView` and `StatusBar`
* This must be a single Book layout, not a list

```
const book = BOOKS_MOCK[X];
<>
  <StatusBar barStyle="dark-content" />
  <SafeAreaView> => CHECK STYLES here!
    <View style={styles.wrapper}> => And here!
      <BookCard imageUrl={book.imageUrl} title={book.title} author={book.author} />
    </View>
  </SafeAreaView>
</>
```